homeassistant/components/media_player/epson.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, websession, name, host, port, encryption):
         self._volume = None
         self._state = None
 
-    async def update(self):
+    async def async_update(self):
```

Comment:
Before this change, since this method was named `update` while being a coroutine, the method would never be executed. That's probably the reason for the behavior you've noticed.